docs/Create-Packages/Creating-a-Package.md

```diff
@@ -60,6 +61,8 @@ Most general-purpose packages contain one or more assemblies that other develope
 
 Resources are, in fact, a special case. When a package is installed into a project, NuGet automatically adds assembly references to the package's DLLs, *excluding* those that are named `.resources.dll` because they are assumed to be localized satellite assemblies (see [Creating localized packages](creating-localized-packages.md)). For this reason, avoid using `.resources.dll` for files that otherwise contain essential package code.
 
+If your library contains COM interop assemblies, you need to follow additional authoring guidelines to make them compatible with package reference (see [Authoring COM Interop packages compatible with Package Reference](#authoring-com-interop-packages-compatible-with-package-reference) )
```

Comment:
Same as above, guidelines should be you need to do following in order to include interop assemblies in your package irrespective of the fact of PackageReference or packages.config